scripts/gaussian3d.py

```diff
@@ -1,4 +1,5 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
+# Author: Mathieu Guay-Paquet
```

Comment:
Most of the code is copy-pasted from astropy, so this should probably be:
```
# Author: Astropy Developers, Mathieu Guay-Paquet
```
(See the original [LICENSE.rst](https://github.com/astropy/astropy/blob/main/LICENSE.rst).)

Reply:
fixed in 0324a3474e99df918bdc6ee6b64aa1574e22ef2e